### ISSUE TITLE
not everthing have to be about the war

### ISSUE THREAD

**Issue by vexii** [facebookexperimental/Recoil#1672]
i do wish and feel for the people directry affcted by this horrible situation.  
But i feel link i should be able to look up tech documentation without being reminded of every evil thing going on in the world. each time i read that banner i compleatly lose focus and start considering how my friends are doing. and i don't feel like this is the responsebility of the recoil website. it also brings me to qustion why something like this is not done for Yemen?  
  
TLDR:  
can we keep the banner to maybe just the frontpage and keep the documentation focused on documentation so that are easily distracted can get there work done?

**Comment by sawariz0r** [comment]
That's the whole point. To remind us that there's a war going on with an aggressor that has threatened your neighbouring countries and has potential to bring war to the nordics as well. 

So if it's bothering you too much, use an adblocker to block out the element on the page. But I vote the banner stays.

**Comment by vexii** [comment]
i think there is a time and place for this kind of stuff. and let's admit it. if your are constantly reminded of human suffering it affects your mood. i don't think software projects should aim to depress and distract developers. no matter who they are or where they are from. why can't people that likes this kind of constant reminders just add a post'it to there monitor with some thing depressing?

**Comment by sawariz0r** [comment]
The time and place is literally now. We can't close our eyes and pretend it's not within our borders and that our lives are not and will not be affected. And if you think anyone likes these reminders - I've got bad news for you; we don't like them either. 

And as I said, if it bothers you and distracts you so much - please consider blocking the element with an ad blocker and continue your ignorance.

**Comment by sawariz0r** [comment]
1. I answered this several times already. 
2. Just opt out.

**Comment by vexii** [comment]
1. no you didn't you just said that we need to be reminded about this all the time. but never explained why being sad 24/7 is better.  
2. again why cant it be opt in? it's objectively a negative and depressing thing so the people that likes that kind of thing should opt in, you don't even need a extension for it. just print something picture or write it on a post-it? but i would like to not be distracted for 20 min each time i am trying to look of the signature for `recoilCallback`.

**Comment by dmitryvinn-fb** [comment]
Here at Meta Open Source, we aim to make our communities welcoming for all, and you could read more about our response in https://github.com/reactjs/reactjs.org/issues/4433#issuecomment-1060759889
